dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -0,0 +1,2118 @@
+
+        Item author1 = ItemBuilder.createItem(context, col1)
+                                  .withTitle("Author1")
+                                  .withIssueDate("2017-10-17")
+                                  .withAuthor("Smith, Donald")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item publication = ItemBuilder.createItem(context, col3)
+                                      .withTitle("Publication1")
+                                      .withAuthor("Testy, TEst")
+                                      .withIssueDate("2015-01-01")
+                                      .withRelationshipType("Publication")
+                                      .build();
+
+        RelationshipType isAuthorOfPublicationRelationshipType = relationshipTypeService
+            .findbyTypesAndLabels(context, entityTypeService.findByEntityType(context, "Publication"),
+                                  entityTypeService.findByEntityType(context, "Person"),
+                                  "isAuthorOfPublication", "isPublicationOfAuthor");
+
+
+
+        EPerson user = ePersonService.create(context);
+        user.setFirstName(context, "first");
+        user.setLastName(context, "last");
+        user.setEmail("testazeazeazezae@email.com");
+        user.setCanLogIn(true);
+        user.setLanguage(context, I18nUtil.getDefaultLocale().getLanguage());
+        ePersonService.setPassword(user, password);
+        // actually save the eperson to unit testing DB
+        ePersonService.update(context, user);
+        context.setCurrentUser(user);
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(user.getEmail(), password);
+
+        MvcResult mvcResult = getClient(token).perform(post("/api/core/relationships")
+                                                           .param("relationshipType",
+                                                                  isAuthorOfPublicationRelationshipType.getID()
+                                                                                                       .toString())
+                                                           .contentType(MediaType.parseMediaType
+                                                               (org.springframework.data.rest.webmvc.RestMediaTypes
+                                                                    .TEXT_URI_LIST_VALUE))
+                                                           .content(
+                                                               "https://localhost:8080/spring-rest/api/core/items/" + publication.getID() + "\n" +
+                                                                   "https://localhost:8080/spring-rest/api/core/items/" + author1.getID()))
+                                              .andExpect(status().isForbidden())
+                                              .andReturn();
+
+    }
+
```

Comment:
@abollini : I'm not sure that test would do anything new, unless I'm misunderstanding your request.  Since a POST always takes the first URI as the "leftItem" and the second URI as the "rightItem", the test you are suggesting should be identical to the existing `createRelationship*` tests.

Reply:
If I understood your explanation the test with the URIs in the inverse order should fail as in the inverse order, author1 will be not a valid item for the leftItem part of the relationship and publication will be not a valid rightItem